h5p-bildetema/src/components/Header/Header.tsx

```diff
@@ -1,20 +1,40 @@
 import React from "react";
 import { TopicGridSizes } from "../../../../common/types/types";
-import { Breadcrumbs } from "../Breadcrumbs/Breadcrumbs";
 import { TopicSizeButtons } from "../TopicSizeButtons/TopicSizeButtons";
+import { Toggle, Breadcrumbs } from "..";
 import styles from "./Header.module.scss";
 
 export type HeaderProps = {
   currentLanguageCode: string;
   topicsSize: TopicGridSizes;
   setTopicsSize: React.Dispatch<React.SetStateAction<TopicGridSizes>>;
+  isWordView: boolean;
+  toggleChecked: boolean;
+  handleToggleChange: (value: boolean) => void;
 };
 
 export const Header: React.FC<HeaderProps> = ({
   currentLanguageCode,
   topicsSize,
   setTopicsSize,
+  isWordView,
+  toggleChecked,
+  handleToggleChange,
 }) => {
+  const renderLeftMenu = (): JSX.Element => {
+    const element = isWordView ? (
+      <Toggle
+        label="Vis skrevne ord"
```

Comment:
We should translate this label (add it to the l10n section in semantics.ts and use `useL10n` to get the translation in the current site language)